### PR TITLE
Incorporate check of #1364

### PR DESCRIPTION
**Pull request by hpohekar**
Incorporated check of https://github.com/pyansys/pyfluent/issues/1364

Created `tests/test_fluent_fixes.py` to track fluent fixes for reference in future.

### CODE REVIEW

File: tests/test_fluent_fixes.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+        }
+    )
+
+    assert solver.solution.report_definitions.volume[
+        "xxx"
+    ].zone_names.allowed_values() == ["fluid"]
+
+    assert (
+        solver.solution.report_definitions.volume["xxx"].expr_list.allowed_values()
+        == None
```
[the remark this answers is not in the document]
Comment (hpohekar) [reply tests/test_fluent_fixes.py:36]: @mkundu1 

You're right. It should return an empty list. 

image

Earlier I compared against an empty list but it was failing and it returns `None`.

image

**Review by prmukherj — APPROVED** [review]

File: tests/test_fluent_fixes.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (mkundu1) [reply tests/test_fluent_fixes.py:36]: Sorry, I don't fully follow. Does that mean the unittest is not testing the correct behaviour? Do we need to fix some code?

[the remark this answers is not in the document]
Comment (hpohekar) [reply tests/test_fluent_fixes.py:36]: You're right. I think @seanpearsonuk can give more guidance on this.

[the remark this answers is not in the document]
Comment (hpohekar) [reply tests/test_fluent_fixes.py:36]: @seanpearsonuk Are we checking correct behavior here ?

[the remark this answers is not in the document]
Comment (seanpearsonuk) [reply tests/test_fluent_fixes.py:36]: @mkundu1 @hpohekar does this mean that for some parameters the API itself is defining allowed-values as None?

[the remark this answers is not in the document]
Comment (hpohekar) [reply tests/test_fluent_fixes.py:36]: @seanpearsonuk I think yes, otherwise it should have returned an empty list as per the fix provided.

**Comment by hpohekar** [comment]
@seanpearsonuk Shall we close this PR and related issue ?

**Review by seanpearsonuk — APPROVED** [review]

**Pull request merged**